cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -31,15 +31,25 @@
 )
 from cg.store.models import IlluminaFlowCell, IlluminaSequencingRun
 from cg.store.store import Store
+from cg.utils.files import get_directories_in_path
 
 LOG = logging.getLogger(__name__)
 
 
 class IlluminaPostProcessingService:
-    def __init__(self, status_db: Store, housekeeper_api: HousekeeperAPI, dry_run: bool) -> None:
+    def __init__(
+        self,
+        status_db: Store,
+        housekeeper_api: HousekeeperAPI,
+        demultiplexed_runs_dir: Path,
+        dry_run: bool,
+        force: bool,
+    ) -> None:
         self.status_db: Store = status_db
         self.hk_api: HousekeeperAPI = housekeeper_api
+        self.demultiplexed_runs_dir = demultiplexed_runs_dir
         self.dry_run: bool = dry_run
```

Comment:
The dry run is only used in the endpoint function and not propagated around, maybe it does not need to be an attibute, just a parameter of `post_process_illumina_flow_cell`

Reply:
will check in new pr